### PR TITLE
Update baseline markdowns with some of the "structural" changes to support automation and "substantive" change efforts

### PR DESCRIPTION
**Pull request by ahuynhMITRE**
## 🗣 Description ##

Updating the baseline markdown files with some of the identified  "structural" changes to serve as a starting point for the team to start on "substantive" changes and in support of the automation of the markdown files to json (issue #209). Following changes include:

- REMOVE All Numbering from “Introductions”, “Assumptions”, and “Baseline” sections of the markdown. 
- ADD New baseline versioning schema to each policy.
- ADD “Last Modified” section below “Rationale” Section.
- UPDATE Group Name to remove "SHALL" and "SHOULD" language.
- UPDATE all relative references (e.g., "see above policies”) with new policy IDs.
- updated all policy IDs for exchange from MS.EXCHANGE to MS.EXO
- Updated the markdown file name exchange.md to exo.md

## 💭 Motivation and context ##

This change is required to support testing of Issue #209 and is in support of creating a "starting point" for the issues associated with the "substantive" changes for each M365 product. Many of the updates also support partially resolving the "Markdown document alignment issues"  opened for each product. 

## 🧪 Testing ##

Changes can be tested by confirming script used in Issue #209 is supported. Markdown files are not corrupt and open as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: baselines/sharepoint.md [hunk 1]
```diff
@@ -64,29 +64,31 @@ or [G3](https://www.microsoft.com/en-us/microsoft-365/government)
 license level. Therefore, only licenses not included in E3/G3 are
 listed.
 
-# 2. Baselines
+# Baselines
 
-## 2.1 File and Folder Links Default Sharing Settings SHALL Be Set to "Specific People (Only the People the User Specifies)"
+## 1 File and Folder Links Default Sharing Settings
```
Comment (crutchfield) [inline baselines/sharepoint.md:69]:
```suggestion
## 1. File and Folder Links Default Sharing Settings
```

File: baselines/sharepoint.md [hunk 2]
```diff
@@ -96,7 +98,7 @@ In the **SharePoint admin** **center**:
 2.  Under **File and folder links**, ensure that the default link type
     is set to **Specific people (only the people the user specifies)**.
 
-## 2.2 External Sharing SHOULD be Set to “New and Existing Guests” and Managed Through Approved Domains and/or Security Groups Per Interagency Collaboration Needs.
+## 2 External Sharing
```
Comment (crutchfield) [inline baselines/sharepoint.md:101]:
```suggestion
## 2. External Sharing
```

File: baselines/sharepoint.md [hunk 3]
```diff
@@ -157,7 +160,7 @@ To adjust sharing settings, in the **SharePoint admin center**:
 
 11. Select **Save.**
 
-## 2.3 Sensitive SharePoint Sites SHOULD Adjust Their Default Sharing Settings to Those Best Aligning to Their Sensitivity Level 
+## 3 Sensitivity Levels
```
Comment (crutchfield) [inline baselines/sharepoint.md:163]:
```suggestion
## 3. Sensitivity Levels 
```

Comment (crutchfield) [inline baselines/sharepoint.md:163]:
```suggestion
## 3. Sensitivity Levels 
```

File: baselines/sharepoint.md [hunk 4]
```diff
@@ -201,7 +205,7 @@ To limit external sharing by domain, in the **SharePoint admin center**:
 
 8.  Select **Save.**
 
-## 2.4 Expiration Times for Guest Access to a Site or OneDrive, and Reauthentication Expiration Times for People Who Use a Verification Code, SHOULD Be Determined By Mission Needs / Agency Policy or Else Defaulted to 30 Days.
+## 4 Guest Access to a Site or OneDrive
```
Comment (crutchfield) [inline baselines/sharepoint.md:208]:
```suggestion
## 4. Guest Access to a Site or OneDrive
```

File: baselines/sharepoint.md [hunk 5]
```diff
@@ -245,26 +253,26 @@ To limit external sharing by domain, in the **SharePoint admin center**:
 
 6.  Enter “30 days”.
 
-## 2.5 Users SHALL Be Prevented from Running Custom Scripts
+## 5 Custom Scripts
```
Comment (crutchfield) [inline baselines/sharepoint.md:256]:
```suggestion
## 5. Custom Scripts
```

File: baselines/aad.md [hunk 6]
```diff
@@ -1232,32 +1281,38 @@ of the tenant):
...
 
-## 2.18 Guest User Access SHOULD Be Restricted
+## 18. Guest User Access
 
 Ensure that only users with specific privileges can invite guest users
 to the tenant and that invites can only be sent to specific external
 domains. Also ensure that guest users have limited access to Azure AD
 directory objects.
 
 ### 2.18.1 Policy
```
Comment (Sloane4) [inline baselines/aad.md:1291]: Forgot to remove 2.18.1 from policy section header

File: baselines/defender.md [hunk 7]
```diff
@@ -59,50 +59,61 @@ the use of the preset profiles. Instead, it enumerates all relevant
...
+### Resources
 
 - [Recommended settings for EOP and Microsoft Defender for Office 365
   security \| Microsoft
   Docs](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365?view=o365-worldwide#eop-anti-spam-policy-settings)
 
-### 2.1.3 License Requirements
+### License Requirements
 
 - N/A
```
Comment (Sloane4) [inline baselines/defender.md:77]: Is there supposed to be an implementation header for this policy

File: baselines/exo.md [hunk 8]
```diff
@@ -188,27 +198,36 @@ Docs](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-securit
 8.  Return to the DKIM page on the Defender admin center to finish
     enabling DKIM.
 
-## 2.4 Domain-Based Message Authentication, Reporting, and Conformance SHALL Be Enabled
+## 4 Domain-Based Message Authentication, Reporting, and Conformance (DMARC)
```
Comment (Sloane4) [inline baselines/exo.md:201]: Missing . after 4, 4 -> 4.

File: baselines/exo.md [hunk 9]
```diff
@@ -404,22 +436,23 @@ Minimum Viable Secure Configuration Baseline*. The DLP solution selected
...
+#### MS.EXO.8.2v1
+The DLP solution SHALL protect PII and sensitive information, as defined by the agency. At a minimum, the sharing of credit card numbers, Taxpayer Identification Numbers (TIN), and Social Security Numbers (SSN) via email SHALL be restricted.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
-### 2.8.2 Resources
+### Resources
 
 - The “Data Loss Prevention SHALL Be Enabled” section of the *Defender
   for Office 365 Minimum Viable Secure Configuration Baseline*.
```
Comment (Sloane4) [inline baselines/exo.md:454]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 10]
```diff
@@ -436,25 +469,29 @@ Be Enabled” section of the Defender for Office 365 Minimum Viable Secure
...
+Disallowed file types SHALL be determined and set. At a minimum, click-to-run files SHOULD be blocked (e.g., .exe, .cmd, and .vbe).
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
-### 2.9.2 Resources
+### Resources
 
 - The “Common Attachments Filter SHALL Be Enabled” section of the
   *Defender for Office 365 Minimum Viable Secure Configuration
   Baseline*.
```
Comment (Sloane4) [inline baselines/exo.md:493]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 11]
```diff
@@ -483,7 +528,7 @@ Office 365 Minimum Viable Secure Configuration Baseline*:
   *Defender for Office 365 Minimum Viable Secure Configuration
   Baseline.*
```
Comment (Sloane4) [inline baselines/exo.md:530]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 12]
```diff
@@ -505,22 +550,28 @@ phishing protections with Microsoft’s native solutions, see the
...
+- _Last modified:_ June 2023
 
-- The phishing protection solution SHOULD include an AI-based phishing
-  detection tool comparable to EOP Mailbox Intelligence.
-
-### 2.11.2 Resources
+### Resources
 
 - The “Phishing Protections SHOULD Be Enabled” section of the *Defender
   for Office 365 Minimum Viable Secure Configuration Baseline.*
```
Comment (Sloane4) [inline baselines/exo.md:573]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 13]
```diff
@@ -645,24 +706,30 @@ this baseline setting may be used. See the “Inbound Anti-Spam
...
+Allowed senders MAY be added, but allowed domains SHALL NOT be added.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
-### 2.14.2 Resources
+### Resources
 
 - The “Inbound Anti-Spam Protections SHALL Be Enabled” section of the
   *Defender for Office 365 Minimum Viable Secure Configuration
   Baseline*.
```
Comment (Sloane4) [inline baselines/exo.md:731]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 14]
```diff
@@ -688,20 +755,29 @@ guidance for enabling link scanning using Microsoft Defender is included
...
+#### MS.EXO.15.3v1
+User click tracking SHOULD be enabled.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
-### 2.15.2 Resources
+### Resources
 
 - The “Safe Links Policies SHOULD Be Enabled” section of the *Defender
   for Office 365 Minimum Viable Secure Configuration Baseline*.
```
Comment (Sloane4) [inline baselines/exo.md:779]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 15]
```diff
@@ -731,16 +807,20 @@ Secure Configuration Baseline*.
...
+#### MS.EXO.16.2v1
+The alerts SHOULD be sent to a monitored address or incorporated into a SIEM.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
-### 2.16.2 Resources
+### Resources
 
 - The “Alerts SHALL Be Enabled” section of the *Defender for Office 365
   Minimum Viable Secure Configuration Baseline*.
```
Comment (Sloane4) [inline baselines/exo.md:822]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/exo.md [hunk 16]
```diff
@@ -757,17 +837,23 @@ guidance configuring audit logging, see the “Audit Logging SHALL Be
...
+#### MS.EXO.17.3v1
+Audit logs SHALL be maintained for at least the minimum duration dictated by [OMB M-21-31 (Appendix C)](https://www.whitehouse.gov/wp-content/uploads/2021/08/M-21-31-Improving-the-Federal-Governments-Investigative-and-Remediation-Capabilities-Related-to-Cybersecurity-Incidents.pdf).
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
-### 2.17.2 Resources
+### Resources
 
 - The “Unified Audit Logging SHALL Be Enabled” section of the *Defender for
   Office 365 Minimum Viable Secure Configuration Baseline*.
```
Comment (Sloane4) [inline baselines/exo.md:859]: Is this supposed to be missing License Requirements & Implementation section?

File: baselines/defender.md [hunk 17]
```diff
@@ -289,7 +305,7 @@ updated malware classification rule).
 - Requires Defender for Office 365 Plan 1 or 2. These are included with
```
Comment (Sloane4) [inline baselines/defender.md:305]: Forgot to remove 2.4.3 from License Requirements

File: baselines/defender.md [hunk 18]
```diff
@@ -649,20 +738,28 @@ learning and analysis techniques to detect malicious intent. Enabling
 this feature may slow down message delivery to the user due to the
 scanning.
 
-### 2.8.1 Policy
+### Policies
```
Comment (Sloane4) [inline baselines/defender.md:741]: Since Policy is a section header should it be consistent through the document as Policy or Policies?

**Review by Sloane4 — CHANGES_REQUESTED** [review]

File: baselines/aad.md [hunk 19]
```diff
@@ -642,26 +678,28 @@ frequency,
...
+## 10. Browser Sessions
 
 To reduce the risk of credential theft during user sessions, disallow
 persistent browser sessions.
 
-### 2.10.1 Policy
-
+### Policy
+#### MS.AAD.10.1v1
 - Browser sessions SHALL not be persistent.
```
Comment (crutchfield) [inline baselines/aad.md:688]:
```suggestion
Browser sessions SHALL not be persistent.
```

File: baselines/aad.md [hunk 20]
```diff
@@ -754,21 +793,20 @@ duties instead of being assigned the Global Administrator role.
...
 
-- Users that need to be assigned to highly privileged Azure AD roles
-  SHALL be provisioned cloud-only accounts that are separate from the
-  on-premises directory or other federated identity providers.
-
-- The following built-in Azure AD roles are considered highly privileged
-  at a minimum. Additional built-in roles that are considered highly
-  privileged in the agency’s environment can be added to this list:
+#### MS.AAD.12.2v1
+The following built-in Azure AD roles are considered highly privileged at a minimum. Additional built-in roles that are considered highly privileged in the agency’s environment can be added to this list:
```
Comment (crutchfield) [inline baselines/aad.md:809]:
```suggestion
The following built-in Azure AD roles are considered highly privileged at a minimum. Additional built-in roles that are considered highly privileged in the agency's environment can be added to this list:
```

File: baselines/defender.md [hunk 21]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
 impersonation attempt, the email is quarantined. Mailbox intelligence is
 an AI-based tool for identifying potential impersonation attempts.
 
-### 2.5.1 Policy
-
+### Policies
+#### MS.DEFENDER.5.1v1
 - User impersonation protection SHOULD be enabled for key agency
```
Comment (crutchfield) [inline baselines/defender.md:341]:
```suggestion
User impersonation protection SHOULD be enabled for key agency
```

File: baselines/defender.md [hunk 22]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
   leaders.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.2v1
 - Domain impersonation protection SHOULD be enabled for domains owned by
```
Comment (crutchfield) [inline baselines/defender.md:347]:
```suggestion
Domain impersonation protection SHOULD be enabled for domains owned by
```

File: baselines/defender.md [hunk 23]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
   the agency.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.3v1
 - Domain impersonation protection SHOULD be added for frequent partners.
```
Comment (crutchfield) [inline baselines/defender.md:353]:
```suggestion
Domain impersonation protection SHOULD be added for frequent partners.
```

File: baselines/defender.md [hunk 24]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.4v1
 - Trusted senders and domains MAY be added in the event of false
```
Comment (crutchfield) [inline baselines/defender.md:358]:
```suggestion
Trusted senders and domains MAY be added in the event of false
```

File: baselines/defender.md [hunk 25]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
   positives.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.5v1
 - Intelligence for impersonation protection SHALL be enabled.
```
Comment (crutchfield) [inline baselines/defender.md:364]:
```suggestion
Intelligence for impersonation protection SHALL be enabled.
```

File: baselines/defender.md [hunk 26]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.6v1
 - Message action SHALL be set to quarantine if the message is detected
```
Comment (crutchfield) [inline baselines/defender.md:369]:
```suggestion
Message action SHALL be set to quarantine if the message is detected
```

File: baselines/defender.md [hunk 27]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
   as impersonated.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.7v1
 - Mail classified as spoofed SHALL be quarantined.
```
Comment (crutchfield) [inline baselines/defender.md:375]:
```suggestion
Mail classified as spoofed SHALL be quarantined.
```

File: baselines/defender.md [hunk 28]
```diff
@@ -320,26 +336,47 @@ the sender address is significantly similar, as to indicate an
...
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.8v1
 - All safety tips SHALL be enabled, including:
```
Comment (crutchfield) [inline baselines/defender.md:380]:
```suggestion
All safety tips SHALL be enabled, including:
```

File: baselines/defender.md [hunk 29]
```diff
@@ -350,22 +387,27 @@ an AI-based tool for identifying potential impersonation attempts.
...
-  - “?” for unauthenticated senders for spoof, and
+  - ? for unauthenticated senders for spoof, and
 
-  - “via” tag.
+  - via tag.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.5.9v1
 - The above configurations SHALL be set in the default policy and SHOULD
```
Comment (crutchfield) [inline baselines/defender.md:397]:
```suggestion
The above configurations SHALL be set in the default policy and SHOULD
```

File: baselines/defender.md [hunk 30]
```diff
@@ -548,28 +612,53 @@ scanning service. Their proxy can perform the following:
...
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+
+#### MS.DEFENDER.7.7v1
+User click tracking SHALL be enabled.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+
+#### MS.DEFENDER.7.8v1
 - Safe Links in Office 365 apps SHALL be turned on.
```
Comment (crutchfield) [inline baselines/defender.md:652]:
```suggestion
Safe Links in Office 365 apps SHALL be turned on.
```

File: baselines/defender.md [hunk 31]
```diff
@@ -548,28 +612,53 @@ scanning service. Their proxy can perform the following:
...
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
 
+#### MS.DEFENDER.7.9v1
 - Users SHALL NOT be enabled to click through to the original URL.
```
Comment (crutchfield) [inline baselines/defender.md:657]:
```suggestion
Users SHALL NOT be enabled to click through to the original URL.
```

File: baselines/onedrive.md [hunk 32]
```diff
@@ -104,22 +107,28 @@ can help avoid unwanted changes. If Anyone links are enabled, the
 expiration date SHOULD be set to thirty days or as determined by mission
 needs or agency policy.
 
-### 2.2.1 Policy
+### Policies
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/onedrive.md:110]: will make all headers "Policies" across all baselines

File: baselines/onedrive.md [hunk 33]
```diff
@@ -173,7 +185,7 @@ content, consider setting the file and folder permissions to **View**.
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/onedrive.md:185]: updated!

File: baselines/onedrive.md [hunk 34]
```diff
@@ -183,27 +195,29 @@ content, consider setting the file and folder permissions to **View**.
 3.  Under **Advanced settings for Anyone links**, set the file and
     folder permissions to **View**.
 
-## 2.4 OneDrive Client SHALL Be Restricted to Windows for Agency-Defined Domain(s)
+## 4. OneDrive Client
 
-Configuring OneDrive to sync only to agency-defined domains ensures that
+Configuring OneDrive to sync only to agency-deined domains ensures that
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/onedrive.md:200]: updated!

File: baselines/powerplatform.md [hunk 35]
```diff
@@ -273,24 +280,33 @@ external tenants) cross-tenant connections are blocked by Power Platform
 even if the user presents valid credentials to the Azure AD-secured data
 source.
 
-### 2.3.1 Policy
+### Policies
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/powerplatform.md:283]: will make all headers "Policies" across all baselines

File: baselines/powerplatform.md [hunk 36]
```diff
@@ -320,21 +336,22 @@ to apply this setting. Also, there is no current way to implement this
 setting for Canvas Apps. When enabled, this setting will apply to all
 current Model-driven apps at only the environment level.
 
-### 2.4.1 Policy
-
-- Content security policies for model-driven Power Apps SHALL be
-  enabled.
+### Policy
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/powerplatform.md:339]: will make all headers "Policies" across all baselines

**Review by crutchfield — CHANGES_REQUESTED** [review]
A few minor corrections to markdowns.  All are in-line so you can just accept if you agree. Then I'll approve.

**Review by adhilto — APPROVED** [review]
I don't have anything to add, looks good to me

**Review by crutchfield — APPROVED** [review]

**Review by tkol2022 — APPROVED** [review]

**Review by Dylan-MITRE — APPROVED** [review]

File: baselines/aad.md [hunk 37]
```diff
@@ -158,25 +164,27 @@ administrator to re-enable their access.
...
-## 2.3 High Risk Sign-ins SHALL Be Blocked
+## 3. High Risk Sign-ins
 
 Azure AD Identity Protection uses various signals to detect the risk
 level for each user sign-in. Sign-ins detected as high risk are to be
 blocked via Conditional Access.
 
-### 2.3.1 Policy
-
+###  Policy
```
Comment (Sloane4) [inline baselines/aad.md:181]: This should be policies

File: baselines/aad.md (hunk 6 above)
Comment (Sloane4) [reply baselines/aad.md:1291]: needs to be changed to policies as well

File: baselines/exo.md [hunk 38]
```diff
@@ -436,25 +469,29 @@ Be Enabled” section of the Defender for Office 365 Minimum Viable Secure
...
 
-### 2.9.1 Policy
-
-- Emails SHALL be filtered by the file types of included attachments.
-  The selected filtering solution SHOULD offer services comparable to
-  Microsoft Defender’s Common Attachment Filter.
+### Policies
+#### MS.EXO.9.1v1
+Emails SHALL be filtered by the file types of included attachments. The selected filtering solution SHOULD offer services comparable to Microsoft Defenders Common Attachment Filter.
+- _Rationale:_ TODO
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/exo.md:475]: sounds good, good catch david!

File: baselines/defender.md [hunk 39]
```diff
@@ -143,7 +154,7 @@ information is sensitive and how that information is handled.
 5.  Select **Edit policy**.
 
 6.  Edit the name and description of the policy if desired, then click
-    **Next**. 
+    **Next**.
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/defender.md:157]: probably, will update

File: baselines/aad.md (hunk 37 above)
Comment (ahuynhMITRE) [reply baselines/aad.md]: updated!

**Comment by nanda-katikaneni** [comment]
An updated/new PR has been approved and merged into emerald - that merged pr has all the structural changes and review suggestions. Closing this pr without merge.

**Pull request closed without merging**